iib/workers/tasks/build_merge_index_image.py

```diff
@@ -232,6 +232,12 @@ def handle_merge_request(
         missing_bundle_paths = [bundle['bundlePath'] for bundle in missing_bundles]
         if missing_bundle_paths:
             add_max_ocp_version_property(missing_bundle_paths, temp_dir)
+            # we have to build and push image to external registry because we have to use it
+            # when we deprecate bundles
+            # Note: since we modified only index.db LABELs are alraedy present in Dockerfile
```

Comment:
TYPO: alraedy

Reply:
So you say that because we didn't modify LABELs but how is it connected to index.db?

Reply:
I mean when we build the image we add LABELs to Dockerfile. 
The labels are  'com.redhat.index.delivery.version' and 'com.redhat.index.delivery.distribution_scope'.

And since we did not generated new docker file we do not have to add those labels as we do before every build.